docs/user/ppl/cmd/patterns.md

```
@@ -0,0 +1,260 @@
+* new_field: optional. Alias of the output pattern field. **Default:** "patterns_field".  
+* algorithm parameters: optional. Algorithm-specific tuning:  
+  * `simple_pattern`: Define regex via "pattern".  
+  * `brain`: Adjust sensitivity with variable_count_threshold and frequency_threshold_percentage.  
+    * `variable_count_threshold`: optional integer. Words are split by space. Algorithm counts how many distinct words are at specific position in initial log groups. Adjusting this threshold can determine the sensitivity of constant words. **Default:** 5.  
+    * `frequency_threshold_percentage`: optional double. Brain's log pattern is selected based on longest word combination. This sets the lower bound of frequency to ignore low frequency words. **Default:** 0.3.  
+  
+## Change the default pattern method  
+
+To override default pattern parameters, users can run following command
+  
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)